app/javascript/components/pages/RecipeIndex.js

```diff
@@ -6,31 +6,41 @@ export default class RecipeIndex extends Component {
     super(props);
     this.state = {
       searchResults: [],
+      pantry: [],
     };
   }
 
-  getRecipes = (pantry) => {
-    let ingredientsAfterFirst = pantry.slice(1, pantry.length);
-    let ingWord = ingredientsAfterFirst.map((word) => {
-      return `+${word.name},`;
-    });
-    let text = `${pantry[0].name},${ingWord.join("")}${
-      pantry[pantry.length - 1].name
-    }`;
+  getRecipes = () => {
     fetch(
-      `https://api.spoonacular.com/recipes/findByIngredients?ingredients=${text}&number=10&apiKey=${this.props.api_key}`
+      `http://localhost:3000/ingredients/?user_id=${this.props.current_user.id}`
```

Comment:
You should be able to fetch to `/ingredients/?user_id=${this.props.current_user.id}` since everything is on localhost:3000 right now you don't need to specify.